source/SIL.AppBuilder.Portal.Frontend/src/ui/components/layout/layout.scss

```diff
@@ -0,0 +1,20 @@
+.sidebar {
+  height: 100%;
```

Comment:
depending on your goals with the sidebar, semantic UI's css should already have stuff like this?
(also, I think you'd want `100vh`, cause 100% height can not work sometimes.. I guess still all depending on intent)